Adapter Crash: MeshKit Error

#### Current Behavior
Adapter build and then fails to run successfully.

#### Screenshots/Logs
```
➜  meshery-cilium git:(master) make run
go mod tidy; \
	DEBUG=true GOPROXY=direct GOSUMDB=off go run main.go
INFO[2023-01-15T00:00:36-06:00] Adaptor Listening at port: 10012              app=cilium-adapter
INFO[2023-01-15T00:00:36-06:00] Registering static workloads...               app=cilium-adapter
INFO[2023-01-15T00:00:36-06:00] Components available statically for version v1.13.0-rc4. Skipping dynamic component registeration  app=cilium-adapter
INFO[2023-01-15T00:00:36-06:00] Registering static workloads completed        app=cilium-adapter
INFO[2023-01-15T00:00:36-06:00] lstat /Users/lee/code/meshery-cilium/templates/oam/traits: no such file or directory  app=cilium-adapter
panic: interface conversion: error is *fs.PathError, not *errors.Error

goroutine 11 [running]:
github.com/layer5io/meshkit/errors.GetCode({0x102631440?, 0xc00085fa40?})
	/Users/lee/code/go/bin/pkg/mod/github.com/layer5io/meshkit@v0.6.16/errors/errors.go:89 +0x90
github.com/layer5io/meshkit/logger.(*Logger).Error(0xc0003a80d8, {0x102631440?, 0xc00085fa40})
	/Users/lee/code/go/bin/pkg/mod/github.com/layer5io/meshkit@v0.6.16/logger/logger.go:71 +0x9f
main.registerCapabilities({0x102290e8c, 0x5}, {0x1026552e0, 0xc0003a80d8})
	/Users/lee/code/meshery-cilium/main.go:165 +0x32a
created by main.main
	/Users/lee/code/meshery-cilium/main.go:112 +0x63b
exit status 2
make: *** [run] Error 1
```

#### Environment

- **Host OS:** Mac
- **Meshery Server Version:** stable-v0.6.43
- **Meshery Client Version:** edge

<!-- Optional 
#### To Reproduce
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error
-->

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 📝 [Meshery Adapters Spreadsheet](https://docs.google.com/spreadsheets/d/1rGGpSXC68iDJzNRU-qZIRQsZUwkt1qLFg7JCs4pfvbU/edit?usp=sharing)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

--- a/main.go
+++ b/main.go
@@ -162,7 +162,7 @@ func registerCapabilities(port string, log logger.Handler) {
 	}
 	// Register meshmodel components
 	if err := oam.RegisterMeshModelComponents(instanceID, mesheryServerAddress(), serviceAddress(), port); err != nil {
-		log.Error(err)
+		log.Info(err.Error())
 	}
 }
 

